extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/main/resources/dev-templates/endpoints.html

```diff
@@ -21,10 +28,17 @@ <h4>REST Resources</h4>
  </div>
  {/if}
  {/for}
-</header>
-
-<header>
-
-</header>
+ <h5>Additional Endpoints</h5>
+ {#for additionalEndpoint in info:additionalEndpointInfo}
+ <div class="row">
+  <div class="col">
+   {#if additionalEndpoint.description}
+   {additionalEndpoint.endpoint} &emsp; <a href="{additionalEndpoint.endpoint}">{additionalEndpoint.description}</a>
+   {#else if !endpoint.description}
```

Comment:
A simple `{#else}` should be enough: I don't think `endpoint` exists here, and we already know there's no description due to the `{#if ...` condition.